src/dscanner/main.d

```diff
@@ -197,6 +199,21 @@ else
 		}
 	}
 
+	if (!excludePaths.empty)
+	{
+		args  = expandArgs(args);
```

Comment:
the new args could be double-expanded, which I don't think would usually lead to problems, but is bad style still I think.

Can you add an `else` to this if block and call `args = expandArgs(args)` there, then strip the expandArgs from all further uses below?

Reply:
This would mean that we always expands args, so we can even put this `args = expandArgs(args)` outside an if-else statement I think.